force-app/nksHomePage/lwc/nksHomePageListEntry/nksHomePageListEntry.html

```diff
@@ -57,43 +57,18 @@ <h3>
                                 </lightning-icon>
                             </template>
                             <h3>
-                                <lightning-formatted-url
-                                    class="title"
-                                    value={recordPageUrl}
-                                    label={record.name}
-                                    onclick={navigateToSObject}
-                                ></lightning-formatted-url>
+                                <a href="#" class="title" onclick={navigateToSObject}>{record.name}</a>
```

Comment:
Dette er ikke helt optimalt, et a-tag med onclick har problemer med skjermlesere, onclick trigres aldri når man trykker space (som det skal gjøre for en lenke). Og når href er #, så gir den enda dårligere resultat. Det du burde gjøre er å skrive om navigationMixin til å lage en URL og bruke den direkte i href, og fjærne onclick helt